### PR TITLE
Add sin function to /maths

### PR DESCRIPTION
**Pull request by zer0-x**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

File: maths/sin.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    >>> sin(0.0)
+    0.0
+    >>> sin(90.0)
+    1.0
+    >>> sin(180.0)
+    0.0
+    >>> sin(270.0)
+    -1.0
+    >>> sin(0.68)
+    0.0118679603
```
Comment (poyea) [inline maths/sin.py:32]: but this seems to be incorrect? `sin(0.68) = 0.628793`

**Review by poyea — CHANGES_REQUESTED** [review]
Also, `sin(9999.0) = -0.9876883405951389`?

File: maths/sin.py (hunk 1 above)
Comment (zer0-x) [reply maths/sin.py:32]: The `math.sin` function in python accepts an angle measured in radians, but my sin function accepts an angle measured in degrees.

To get the right result use:
```python
>>> import math
>>> math.sin(math.radians(0.68))
0.011867960298537237
```

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**